zulip/setup.py

```diff
@@ -69,7 +69,8 @@ def recur_expand(target_root: Any, dir: Any) -> Generator[Tuple[str, List[str]],
 setuptools_info = dict(
     install_requires=[
         "requests[security]>=0.12.1",
-        "matrix_client",
+        "matrix-nio",
```

Comment:
Is `requests[security]` still a required dependency? Probably can be removed?

Reply:
oh, right, this is not required any more